src/components/autocomplete-textarea/autocomplete-textarea.tsx

```diff
@@ -25,8 +25,10 @@ function useRules() {
         { regex: /^hei,?$/i, replacement: 'Hei, [bruker.fornavn]\n' },
         {
             regex: /^mvh$/i,
-            replacement: `Med vennlig hilsen\n[saksbehandler.fornavn]\n${
-                erKontaktsenter ? 'NAV Kontaktsenter' : saksbehanderEnhet
+            replacement: `Med vennlig hilsen\n${
```

Comment:
Syns det begynner å bli litt hårete med alle de inline-if'ene; 
Lurer på om vi burde gjøre en liten endring på return-typen til funksjonen for å rydde litt her. 

Forslaget mitt er å endre return-type fra `Array<{ regex: Regexp; replacement: string }>` til `Array<{ regex: Regexp; replacement: () => string }>`. På den måten kan vi skrive denne regel sånn; 

```
{
  regex: /^mvh$/i,
  replacement: () => {
    const mvh = 'Med vennlig hilsen';
    if (erKontaktsenter) {
      return `${mvh}\n[saksbehandler.fornavn]\nNAV Kontaktsenter`
    }
    return `${mvh}\n[saksbehandler.navn]\n${saksbehanderEnhet}`
  }
}
```